### DISCUSSION TITLE
Thread 1.1 - GRL test 5.1.9 inconclusive

### DISCUSSION THREAD

**Discussion by testter2** [openthread/openthread#9880] [Q&A]
Hello,

I have an issue with 5.1.9 test case for router role from GRL which return inconclusive in most trials with the following error

`OT Router could not attach to the network after 20 s of timeout.
`
attached captures for inconclusive and passed captures
[capture for 5_1_9.zip](https://github.com/openthread/openthread/files/14420679/capture.for.5_1_9.zip)

DUT is using latest release from openthread with default configuration OT_THREAD_VERSION_1_3.
GRL v59 is used.

I'm seeking your assistance to get the reason behind this result.

Thanks in advance

**Comment by jwhui** [comment]
Can you share the full test report (including PDF and logs)?

**Comment by testter2** [comment]
@jwhui Attached full logs and report
[5_1_9_inconclusive.zip](https://github.com/openthread/openthread/files/14472207/5_1_9_inconclusive.zip)

I'm seeking your assistance to get the reason behind this result.

Thanks in advance

**Reply by jwhui** [reply]
The Harness Device Information section of the report is empty, which makes it hard to understand the topology and pcap. Are you configuring the Test Harness properly? I would also suggest filing a Thread Group JIRA, if you are having further issues.
